src/Bundle/Resources/config/services/state.xml

```diff
@@ -12,65 +12,30 @@
 -->
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
+        <import resource="state/processor.xml" />
+        <import resource="state/provider.xml" />
+        <import resource="state/**/*.xml" />
+    </imports>
```

Comment:
This is a small thing, but I suggested removing these imports and add a generic one to `services.xml`